easybuild/easyconfigs/f/FFmpeg/FFmpeg-2.8.7-foss-2016a.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'FFmpeg'
+version = '2.8.7'
+
+homepage = 'https://www.ffmpeg.org/'
+description = """A complete, cross-platform solution to record, convert and stream audio and video."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = ['http://ffmpeg.org/releases/']
+
+dependencies = [
+    ('NASM', '2.11.08'),
+    ('zlib', '1.2.8'),
+    ('x264', '20160114'),
```

Comment:
existing easyconfig uses `NASM` 2.12.01 and `x264` 2016114, so bump to those versions here too?